src/commands/codepush/deployment/list.ts

```diff
@@ -27,6 +27,11 @@ export default class CodePushDeploymentListListCommand extends AppCommand {
   @longName("displayKeys")
   public displayKeys: boolean;
 
+  @help("Specifies whether to Fetch Deployment Metric or Not")
+  @shortName("m")
+  @longName("disableDeploymentMetric")
```

Comment:
probably better to name it `skipFetchingDeploymentMetrics`. Short name also should be adjusted, `m` is not a good short name for such parameter.

Reply:
@DmitriyKirakosyan agreed made the changes for long name `skipFetchingDeploymentMetrics` and for short name kept it as `s` and please suggest if you have any other short name

Reply:
I would skip a short name at all, but `s` is fine.